users/ignoring.rst

```diff
@@ -34,62 +34,77 @@ Patterns
-   ``foo``. Spaces are treated as regular characters.
-
--  Asterisk matches zero or more characters in a filename, but does not
-   match the directory separator. ``te*st`` matches ``test``,
-   ``subdir/telerest`` but not ``tele/rest``.
-
--  Double asterisk matches as above, but also directory separators.
-   ``te**st`` matches ``test``, ``subdir/telerest`` and
-   ``tele/sub/dir/rest``.
-
--  Question mark matches a single character that is not the directory
-   separator. ``te??st`` matches ``tebest`` but not ``teb/st`` or
-   ``test``.
-
--  Characters enclosed in square brackets ``[]`` are interpreted as a character range ``[a-z]``. Before using this syntax you should have a basic understanding of regular expression character classes.
-
--  A pattern beginning with ``/`` matches in the current directory only.
-   ``/foo`` matches ``foo`` but not ``subdir/foo``.
-
--  A pattern beginning with ``#include`` results in loading patterns
-   from the named file. It is an error for a file to not exist or be
-   included more than once. Note that while this can be used to include
-   patterns from a file in a subdirectory, the patterns themselves are
-   still relative to the folder *root*. Example:
-   ``#include more-patterns.txt``.
-
--  A pattern beginning with a ``!`` prefix negates the pattern: matching files
-   are *included* (that is, *not* ignored). This can be used to override
-   more general patterns that follow.
-
--  A pattern beginning with a ``(?i)`` prefix enables case-insensitive pattern
-   matching. ``(?i)test`` matches ``test``, ``TEST`` and ``tEsT``. The
-   ``(?i)`` prefix can be combined with other patterns, for example the
-   pattern ``(?i)!picture*.png`` indicates that ``Picture1.PNG`` should
-   be synchronized. On Mac OS and Windows, patterns are always case-insensitive.
-
--  A pattern beginning with a ``(?d)`` prefix enables removal of these files if
-   they are preventing directory deletion. This prefix should be used by any OS
-   generated files which you are happy to be removed.
-
--  A line beginning with ``//`` is a comment and has no effect.
-
--  Windows does not support escaping ``\[foo - bar\]``.
-
-.. note::
-
-   Prefixes can be specified in any order (e.g. "(?d)(?i)"), but cannot be in a
+Regular file names match themselves, i.e. the pattern foo matches the files ``foo``,
+``subdir/foo`` as well as any directory named ``foo``. Spaces are treated as regular characters.
+
++-------------+-----------+----------+-------------------------------------------------------------------------------+
+| Character   | Where     | Affects  | Action                                                                        |
++=============+===========+==========+===============================================================================+
+| ``/``       | Beginning | Folders  | A pattern beginning with ``/`` matches in the current directory only.         |
```

Comment:
Not sure what you mean by "Affects: Folders". And this is the only item that is not either `All` or `N/A`, so I also don't get the point of this column.